airflow/www/templates/airflow/dag.html

```diff
@@ -132,16 +132,21 @@ <h4 class="modal-title" id="taskInstanceModalLabel">
             <hr>
           </div>
           <a id="btn_task" class="btn btn-sm" data-base-url="{{ url_for('Airflow.task') }}">
-            Task Instance Details
+            Instance Details
           </a>
-          <a id="btn_rendered" class="btn btn-sm" data-base-url="{{ url_for('Airflow.rendered') }}">
+          <a id="btn_rendered" class="btn btn-sm" data-base-url="{{ url_for('Airflow.rendered_templates') }}">
             Rendered
           </a>
-          <a id="btn_ti" class="btn btn-sm" data-base-url="{{ url_for('TaskInstanceModelView.list') }}">
-            Task Instances
-          </a>
+          {% if conf_core_executor == 'KubernetesExecutor' %}
```

Comment:
CeleryKubernetesExexcutor too

Reply:
@ryanahamilton how do we do or statements here?

Reply:
{% if conf_core_executor in ['KubernetesExecutor', 'CeleryKubernetesExexcutor'] %}

might work

Reply:
https://jinja.palletsprojects.com/en/2.11.x/templates/

Reply:
I wrote a helper that handled this—this appears to be an old version of the code. Was possibly lost in the squash. I'll see about salvaging it—might have to rewrite. 😞

Reply:
I've added back the missing changes. See `airflow.utils.helpers.k8s_or_k8scelery_executor` for how this was handled.